index.ts

```diff
@@ -16,19 +16,30 @@ function getProgramMap(): Map<String, String> {
 }
 
 export async function main() {
-    const programNameToId = getProgramMap();
+    
+    const cluster = (process.env.CLUSTER || 'localnet') as Cluster;
```

Comment:
It would be good to have a list of env vars currently used with brief explanation to inform use to setup them.